packages/wdio-types/src/Capabilities.ts

```diff
@@ -1087,6 +1091,10 @@ export interface BrowserStackCapabilities {
      * @private
      */
     wdioService?: string
+    /**
+     * @private
+     */
+    'browserstack.wdioService'?: string
```

Comment:
do we need both? `wdioService` and `browserstack.wdioService`?

Reply:
resolved f2f

Reply:
we need `browserstack.wdioService` for when the user has jsonwire caps and we don't want to introduce bstack:options into the user's caps - as that leads to other issues.